engine/gamesys/src/gamesys/components/comp_camera.cpp

```diff
@@ -356,6 +385,11 @@ namespace dmGameSystem
             component->m_OrthographicZoom = params.m_Value.m_Number;
             return dmGameObject::PROPERTY_RESULT_OK;
         }
+        else if (CAMERA_PROP_ASPECT_RATIO == set_property)
```

Comment:
I wonder if we should have a "READ_ONLY" return value 🤔 
"Not found" is a bit misleading

Reply:
Yes, this is a good point. I'll add it.

Reply:
Added!